src/auth/auth.module.ts

```diff
@@ -11,5 +11,6 @@ import * as REDIS_CONFIG from 'src/config/redis';
     TypeOrmModule.forFeature([User]),
   ],
   providers: [AuthResolver, AuthService],
+  exports: [TypeOrmModule],
```

Comment:
Why do you need to export typeorm from auth ? its better to define typeorm module on highest level, app.module for example so you don't need to reexport it